.gitlab-ci.yml

```diff
@@ -34,7 +34,7 @@ lint:
 build:
   artifacts:
     paths:
-      - dist/
+      - *.tgz
```

Comment:
Can we keep dist/ and move files to it in this job after running pack? It's be more consistent with other projects and I feel like building and moving packages to a dir should be in the same job.